tests/foreman/installer/test_installer.py

```diff
@@ -1419,31 +1449,33 @@ def test_positive_check_installer_services():
 
 @pytest.mark.upgrade
 @pytest.mark.tier3
-def test_installer_options_and_flags():
-    """Look for changes on installer options and flags
+@pytest.mark.parametrize('filter', ['params', 'sections'])
+def test_installer_options_and_sections(filter):
+    """Look for changes on installer sections and options/flags
 
     :id: a51d3b9f-f347-4a96-a31a-770349db08c7
 
+    :parametrized: true
```

Comment:
The PR is failing the GHA because of formatting in the function above, but it will also fail testimony validation - this field is a choice of 'yes' or 'no'.

https://github.com/SatelliteQE/robottelo/blob/master/testimony.yaml#L154

Reply:
OOPS ! FIXED!